crates/kitsune_p2p/kitsune_p2p/src/types/event.rs

```diff
@@ -91,7 +92,7 @@ pub struct QueryAgentsEvt {
     /// Optional time range to filter by.
     pub window: Option<TimeWindow>,
     /// Optional arcset to intersect by.
-    pub arc_set: Option<Arc<DhtArcSet>>,
+    pub arc_set: Option<ArqSet>,
```

Comment:
not arq_set?

Reply:
yeah I missed some of these. Do you think it would be good to do a sweep and get them all or can we handle some misnomers? Also did you sweep for all of these, i.e. were there only 4, or were there a lot more?

Reply:
There were a fair amount more, I started from the bottom of the file changes going up and just stopped after 4.

It feels a little weird to have some some way and some the other, but I also don't feel like it should be a priority...